src/gui/update.rs

```diff
@@ -1488,6 +1488,13 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
                 Message::LatestRelease,
             ));
         }
+        Message::Interaction(Interaction::PickGlobalReleaseChannel(channel)) => {
+            log::debug!("Interaction::PickGlobalReleaseChannel({:?})", channel);
+
+            ajour.config.addons.global_release_channel = channel;
+
+            let _ = ajour.config.save();
+        }
```

Comment:
... And if we do `ReleaseChannel::Default(GlobalReleaseChannel)`  what should happen here? Loop and set?